src/components/ui/Mark.tsx

```diff
@@ -0,0 +1,70 @@
+import classnames, {
+  alignItems,
+  borderColor,
+  borderRadius,
+  borderWidth,
+  boxSizing,
+  cursor,
+  display,
+  fontFamily,
+  fontSize,
+  height,
+  justifyContent,
+  width,
+} from 'classnames/tailwind'
+
+const questionStyles = (size?: Sizes, disabled?: boolean) =>
+  classnames(
+    display('flex'),
+    justifyContent('justify-center'),
+    alignItems('items-center'),
+    fontSize(size === Sizes.Small ? 'text-xs' : 'text-base'),
+    fontFamily('font-primary'),
+    cursor(disabled ? undefined : 'cursor-pointer')
+  )
+
+const borderWrapper = (size?: Sizes) => {
+  return classnames(
+    boxSizing('box-content'),
+    borderRadius('rounded-full'),
+    borderWidth('border'),
+    borderColor('border-current'),
+    width({
+      'w-4': size === Sizes.Small,
+      'w-5': size === Sizes.Medium,
+      'w-6': !size,
+    }),
+    height({
+      'h-4': size === Sizes.Small,
+      'h-5': size === Sizes.Medium,
+      'h-6': !size,
+    }),
+    display('flex'),
+    justifyContent('justify-center')
+  )
+}
+export enum MarkType {
+  Question = '?',
+  Exclamation = '!',
+}
+
+export enum Sizes {
+  Small = 'Small',
+  Medium = 'Medium',
+}
+
+export default function ({
```

Comment:
Maybe we should have a better name for this component? Something like `CharInCircle`/`MarkInCircle`?

Reply:
Because mark can be a square or triangle

Reply:
You're right, ty. I'll name it to `CharInCircle`